projects/documentation/content/_includes/component.njk

```diff
@@ -15,6 +15,9 @@ layout: layout.njk
       {% for part in collections[componentName] %}
         <sp-tab-panel class="section" value="{{ part.data.partType }}">
             {{ part.templateContent | safe }}
+            {% if hasReadmePackagesTemplate %}
```

Comment:
This will place this content at the bottom of the Examples _AND_ the API Table which is what is causing duplicate IDs in the same DOM tree and causing the Tooltips not to open in the Examples tab.

Reply:
_Right_. Added the section after this.

Reply:
The logic has been changed to add inside the for loop within the tab panel
```
{% for part in collections[componentName] %}
        <sp-tab-panel class="section" value="{{ part.data.partType }}">
            {{ part.templateContent | safe }}
            {% if part.data.partType === 'examples' %}
              {% set isUnderExamples = true %}
            {% endif %}
              {% if isComponentTemplate and isUnderExamples %}
                  {% include "partials/accessibility.njk" %}
                  {% set isUnderExamples = false %}
              {% endif %}
        </sp-tab-panel>
      {% endfor %}
      
    ```